l5kit/l5kit/environment/callbacks.py

```diff
@@ -0,0 +1,70 @@
+import gym
+from stable_baselines3.common.callbacks import EvalCallback
+
+from l5kit.cle.validators import ValidationCountingAggregator
+from l5kit.environment.gym_metric_set import DisplacementCollisionMetricSet
+
+
+class L5KitEvalCallback(EvalCallback):
+    """Callback for evaluating an agent using L5Kit evaluation metrics.
+
+    :param eval_env: The environment used for initialization
+    :param n_eval_episodes: The number of episodes to test the agent
+    :param eval_freq: Evaluate the agent every ``eval_freq`` call of the callback.
+    :param verbose:
+    :param metric_set: computes a set of metric parametrization for the L5Kit environment
+    """
+
+    def __init__(self, eval_env: gym.Env, eval_freq: int = 10000, n_eval_episodes: int = 10,
+                 n_eval_envs: int = 4, verbose: int = 0) -> None:
+        super(L5KitEvalCallback, self).__init__(eval_env)
+        self.eval_freq = eval_freq
+        self.n_eval_episodes = n_eval_episodes
+        self.n_eval_envs = n_eval_envs
+        self.verbose = verbose
+        self.metric_set = DisplacementCollisionMetricSet()
```

Comment:
If the user wants to change this, it will have to extend this class and change the constructor. Another alternative is to receive this a a parameter.